src/Listener/ViewListener.php

```diff
@@ -12,6 +12,13 @@ class ViewListener extends BaseListener
 {
 
     /**
+     * Default associations config
+     *
+     * @var array
+     */
+    protected $associations = [];
```

Comment:
True, I've been removing those in my local projects and I totally forgot that we still had them here